bittensor/commands/wallets.py

```diff
@@ -827,3 +828,163 @@ def check_config(config: "bittensor.config"):
                 _,
                 config.subtensor.chain_endpoint,
             ) = bittensor.subtensor.determine_chain_endpoint_and_network(str(network))
+
+
+API_URL = "https://api.subquery.network/sq/TaoStats/bittensor-indexer"
```

Comment:
Do we have some metrics on the efficiency and load on this endpoint?  It will be hit quite a lot.

Reply:
Not sure about this. I can perform a basic load test on this using Apache AB. Nonetheless, I am open to suggestions!

Reply:
That should be enough. Doesn't need to be extravagant, just want to get a sense of how much stress this would take before having issues.

Reply:
I am attaching the load test results which contain the below scenarios,

1) Concurrency - 50, Total Requests - 1000
2) Concurrency - 100, Total Requests - 10000
3) Concurrency - 250, Total Requests - 10000

250 is the limit, going above it gives socket error - socket: Too many open files (24)

[load_test.txt](https://github.com/opentensor/bittensor/files/13817638/load_test.txt)